pkgs/top-level/all-packages.nix

```diff
@@ -20400,7 +20401,6 @@ with pkgs;
     else if name == "nblibc" then targetPackages.netbsdCross.libc or netbsdCross.libc
     else if name == "wasilibc" then targetPackages.wasilibc or wasilibc
     else if name == "relibc" then targetPackages.relibc or relibc
-    else if stdenv.targetPlatform.isGhcjs then null
```

Comment:
Was this deletion accidental?  Seems unrelated.

Reply:
No, as I [already explained](https://github.com/NixOS/nixpkgs/pull/228374#discussion_r1178957343), I've removed special handling of GHCJS here, so that other platforms can also have libc = null.